README.markdown

```diff
@@ -1,3 +1,4 @@
+
 # The Official raywenderlich.com Swift Style Guide.
```

Comment:
(Out of scope) - lets change it to **Sweet Studio Style Guide (based on [The Official raywenderlich.com Swift Style Guide](link to official))**